File: PasswordGenerateModel.js
const mongoose = require('mongoose');

const passwordGenerateSchema = new mongoose.Schema({
    includeLatinUppercase: Boolean,
    includeLatinLowercase: Boolean,
    includeCyrillicUppercase: Boolean,
    includeCyrillicLowercase: Boolean,
    includeNumbers: Boolean,
    includeSymbols: Boolean,
    length: Number,
});

const PasswordGenerateModel = mongoose.model('PasswordGenerate', passwordGenerateSchema);

module.exports = PasswordGenerateModel;

File: server_python/password_generate_model.py
from mongoengine import Document, BooleanField, IntField, StringField

class PasswordGenerateModel(Document):
    include_latin_uppercase = BooleanField(default=False)
    include_latin_lowercase = BooleanField(default=False)
    include_cyrillic_uppercase = BooleanField(default=False)
    include_cyrillic_lowercase = BooleanField(default=False)
    include_numbers = BooleanField(default=False)
    include_symbols = BooleanField(default=False)
    length = IntField()
    letters = StringField()